addons/web_unsplash/static/src/js/unsplash_image_widget.js

```diff
@@ -300,6 +300,7 @@ widgetsMedia.ImageWidget.include({
             }
         }
         this._super.apply(this, arguments);
+        this.hasSearched = true;
```

Comment:
Well it is changed... https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/wysiwyg/widgets/media.js#L73